src/http/server.rs

```diff
@@ -159,15 +161,23 @@ impl From<&Configuration> for Newtype<httpd_ssl_config_t> {
         Self(httpd_ssl_config_t {
             httpd: http_config.0,
             session_tickets: false,
+            #[cfg(esp_idf_version_major = "5")]
+            use_secure_element: false,
             port_secure: conf.https_port,
             port_insecure: conf.http_port,
             transport_mode,
             cacert_pem: ptr::null(),
             cacert_len: 0,
             prvtkey_pem: ptr::null(),
             prvtkey_len: 0,
+            #[cfg(esp_idf_version_major = "4")]
             client_verify_cert_pem: ptr::null(),
+            #[cfg(esp_idf_version_major = "4")]
             client_verify_cert_len: 0,
+            #[cfg(esp_idf_version_major = "5")]
+            servercert: ptr::null(),
```

Comment:
renamed in v5.0+
see: https://github.com/espressif/esp-idf/commit/94056fd4a547b63521f9b47fd313b8d1b8a1c7c5#diff-4ac40d704acb6c3ed7b266a6fe23a62209bd770ba94b340d5343381e45980e34L53-L68